tests/slow/src/test/scala/tests/feature/FileDecoderProviderLspSuite.scala

```diff
@@ -26,6 +26,37 @@ class FileDecoderProviderLspSuite extends BaseLspSuite("fileDecoderProvider") {
     Right(FileDecoderProviderLspSuite.tastySingle)
   )
 
+  val userHome: String = System.getProperty("user.home")
+  check(
+    "decode-jar",
+    s"""
+       |/metals.json
+       |{
+       |  "a": {
+       |    "scalaVersion": "${scala.meta.internal.metals.BuildInfo.scala212}",
+       |    "libraryDependencies": [
+       |      "ch.epfl.scala:com-microsoft-java-debug-core:0.21.0+1-7f1080f1"
+       |    ]
+       |  }
+       |}
+       |/a/src/main/scala/a/Main.scala
+       |package a
+       |import com.microsoft.java.debug.core.protocol.Events._
+       |
+       |object Main {
+       |  val a : ExitedEvent = null
+       |  println(a)
+       |}
+       |""".stripMargin,
+    "a/src/main/scala/a/Main.scala",
+    None,
+    "file-decode",
+    Right(FileDecoderProviderLspSuite.EventsJarFile),
+    customUri = Some(
+      s"jar:file://${userHome}/.cache/coursier/v1/https/repo1.maven.org/maven2/ch/epfl/scala/com-microsoft-java-debug-core/0.21.0%2B1-7f1080f1/com-microsoft-java-debug-core-0.21.0%2B1-7f1080f1-sources.jar!/com/microsoft/java/debug/core/protocol/Events.java"
```

Comment:
Maybe I'm missing something, but how does this work locally? For example if I run this test on my local machine, MacOS, this isn't where my coursier cache is located by default?

Reply:
It's not in `userHome` ? I kind of assumed since it passed that my guess was right

Reply:
No, on Mac it's odd, it doesn't go to `$HOME/.cache` but rather `$HOME/Library/Caches`

Reply:
This shouldn't have passed on CI then :sweat_smile:  I am very much confused. 

I will try and fix that.

Reply:
ha that was my thought as well, I don't get how it passes in CI because locally it won't for me.